app/assets/images/payment_icons/latitude_gem_nz.svg

```diff
@@ -0,0 +1 @@
4NkiY9wZzBFJo1uYTXqGWaRcx2RO5WW0590QQxzOTv2pkDQ2R3DFUhC7o887c9OwWI3KuIzIqqDASDAiMxQHp2Z+jT1ox5BBd1h5la6OcY8S5yEqhagEu6kdQJriCMt0i6hYQFpDWdgDwOuV6H4diQ7oNJSCUWZw9HenrdowiWPNbE7TyhojgzwhM0bRRBl4hwwH3GvugWvPDejm3p1JljkEbShW3PeR+ewtSYg7qZDbPVKvP7LMikAlkfkAqkXYZsyt/u3GfjUAIuFlVWZuZeXU/UpWw2uzAlQm5VBeKp/DYCzAdLsDCtlGr2UlWW8LGYi0CC4aubjcSpfTTlVgh+z/AQvzrYpBhUynAAAAAElFTkSuQmCC"/></g><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill="#002a66" fill-rule="evenodd" opacity=".3"/><path d="M12.836 7.628a3.849 3.849 0 00-2.546-.954 3.8 3.8 0 00-3.8 3.8 3.807 3.807 0 003.8 3.8 3.674 3.674 0 002.383-.833 2.853 2.853 0 01-.607.874 2.592 2.592 0 01-3.671 0c-.064-.063-.322-.359-.322-.359l-1.173.425a9.479 9.479 0 00.638.8 3.716 3.716 0 002.7 1.11 3.677 3.677 0 002.685-1.111 3.932 3.932 0 001.09-2.776V7.236h-1.177zm-2.545 5.462a2.607 2.607 0 112.609-2.607 2.607 2.607 0 01-2.609 2.607zm9.309 2.75h-.169v1.181h.857v-.169H19.6zm-1.887 1.017l-.262-.6H17.3l.343.785-.012.027a.233.233 0 01-.06.093.135.135 0 01-.084.029.192.192 0 01-.138-.054l-.06.137a.343.343 0 00.2.061.28.28 0 00.173-.061.369.369 0 00.1-.149l.023-.052.355-.816h-.158zm-1.047-.557a.289.289 0 00-.289-.289h-.531v1.011h.593a.29.29 0 00.289-.289.286.286 0 00-.134-.243.285.285 0 00.072-.19zm-.676-.145h.387a.145.145 0 110 .289h-.387zm.449.723h-.449v-.289h.449a.145.145 0 010 .289zm11.392-.391c0 .213-.085.381-.331.381s-.331-.168-.331-.381v-.647H27v.671a.5.5 0 101 0v-.671h-.168zm1.463-.646h-.486v1.181h.486a.591.591 0 100-1.181zm-.012 1.012h-.3v-.844h.3a.424.424 0 110 .844zm2.231-2.753V9.038a2.066 2.066 0 00-.6-1.511 2.319 2.319 0 00-1.669-.666 2.68 2.68 0 00-1.773.63l-.051.051-.046-.047a2.329 2.329 0 00-1.638-.634 2.825 2.825 0 00-1.643.514.952.952 0 00-.831-.412 1.125 1.125 0 00-.785.3 3.789 3.789 0 00-1.139 2.377v.017A5.7 5.7 0 0120.1 12.31a2.591 2.591 0 01-1.838.755 2.373 2.373 0 01-1.49-.454l4.5-4.5-.307-.336a3.814 3.814 0 00-5.387 0 3.807 3.807 0 000 5.384 3.68 3.68 0 002.682 1.121 3.537 3.537 0 002.7-1.122 7.146 7.146 0 001.531-3.417v-.008c.108-.572.391-1.365.646-1.364.05 0 .2.007.186.426V14.1h1.166V9.055a.974.974 0 01.293-.728 1.622 1.622 0 01.95-.291 1.227 1.227 0 01.814.291.828.828 0 01.282.6V14.1h1.174V9.018a.972.972 0 01.291-.692 1.611 1.611 0 01.948-.29 1.223 1.223 0 01.815.291.839.839 0 01.281.63V14.1zm-15.847-3.63a2.48 2.48 0 01.761-1.836 2.518 2.518 0 011.833-.758 2.407 2.407 0 011.278.311l-3.561 3.561a2.4 2.4 0 01-.311-1.278zm14.99 6.551h.857v-.169h-.688v-.337h.592v-.169h-.592v-.338h.688v-.168h-.857zm-5.388-1.013h.4v1.013h.169v-1.013h.4v-.168h-.977zm-2.667 0h.4v1.013h.169v-1.013h.4v-.168h-.97zm1.738 1.013h.169V15.84h-.169zM21.43 15.84l-.53 1.181h.174l.148-.338h.591l.148.338h.178L21.6 15.84zm-.139.675l.221-.507.222.507z" fill="#fff"/><g clip-path="url(#c)"><g clip-path="url(#d)"><path d="M44.471-29.174h38v23.975h-38z" fill="url(#a)"/></g></g></svg>
```

Comment:
Same for this one

[![alt](https://screenshot.click/08-21-358u6-gbnny.png)](https://screenshot.click/08-21-358u6-gbnny.png)